db/migrate/20220429092052_create_order_cycle_shipping_methods.rb

```diff
@@ -0,0 +1,98 @@
+class CreateOrderCycleShippingMethods < ActiveRecord::Migration[6.1]
+  # Before this migration every available shipping method was available to customers on order
+  # cycles by default. However this migration only populates :order_cycles_shipping_methods records
```

Comment:
Have you thought about a backwards compatible app logic where all methods are selected if none is specifically selected in the database? Then we wouldn't need that flag.